Bugfix from SRC for uninitialized "FS.DirectoryFileType" variable.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/libm3/src/os/Common/m3makefile,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 m3makefile
--- m3/libm3/src/os/Common/m3makefile.orig	1996/09/24 05:21:55	1.1.1.1
+++ m3/libm3/src/os/Common/m3makefile	1996/09/24 05:32:42
@@ -19,6 +19,7 @@
 Interface(Terminal)
 
 /* The following only initialize the File.Type variables: */
+implementation(FS)
 implementation(Terminal)
 implementation(RegularFile)
 implementation(Pipe)
